src/lib/bootloader/grub_install.rb

```diff
@@ -14,11 +14,24 @@ def execute(devices: [], secure_boot: false, trusted_boot: false)
       raise "cannot have secure boot without efi" if secure_boot && !@efi
```

Comment:
np: I would prefer having an `attr_reader :efi` or a `efi?` method instead of use the instance variable.